week1/weekOne-HW/Program.cs

```diff
@@ -0,0 +1,78 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+
+//1. String manipulation
+
+app.MapGet("/Exercise1", (string input) => {
+    char[] charArray = input.ToCharArray();
+    Array.Reverse(charArray);
+    return new string(charArray);
+});
+
+
+
+//2. String/Math
+
+app.MapGet("/Exercise2", (string input) => {
+     char[] vowels = {'a','e','i','o','u'};
+     int vowelCount = 0;
+     string sentence = input.ToLower();
+
+     for (int i = 0; i < sentence.Length; i++)
+     {
+        if (vowels.Contains(sentence[i]))
+        {
+            vowelCount++;
+        }
+        
+     }
+    return $"{input} Contains {vowelCount} vowel";
+
+});
+
+//3. Math/Array
+
+
+app.MapGet("/Exercise3", () => {
+    int[] arr = new[] { 271, -3, 1, 14, -100, 13, 2, 1, -8, -59,  -1852, 41, 5 };
+    int sumOfNegatives = 0;
+    int multiOfPositive =1;
+    for (int i = 0; i < arr.Length; i++)
+    { 
+        if (arr[i]<= 0)
+        {
+            sumOfNegatives -= arr[i];

```

Comment:
Sum of negatives should give negative value but this is ok too 👍

Reply:
I fixed it thank you .